Documentation/building/linux-instructions.md

```diff
@@ -37,7 +40,7 @@ ellismg@linux:~$ sudo apt-get update
 
 Then install the packages you need:
 
-`ellismg@linux:~$ sudo apt-get install cmake llvm-3.5 clang-3.5 lldb-3.6 lldb-3.6-dev libunwind8 libunwind8-dev gettext libicu-dev`
+`ellismg@linux:~$ sudo apt-get install cmake llvm-3.5 clang-3.5 lldb-3.6 lldb-3.6-dev libunwind8 libunwind8-dev gettext libicu-dev curl libcurl4-openssl-dev libssl-dev`
```

Comment:
Looks like I'm wrong, libcurl4-openssl-dev and libssl-dev are needed but not curl.  I'm sure I needed it first time!  

Good spot.

Stephen.